Code/Framework/AzFramework/Platform/iOS/AzFramework/Windowing/NativeWindow_ios.mm

```diff
@@ -52,8 +55,25 @@ void InitWindow(const AZStd::string& title,
         m_nativeWindow = [[UIWindow alloc] initWithFrame: screenBounds];
         [m_nativeWindow makeKeyAndVisible];
 
+#if defined(CARBONATED)
+        m_width = [[UIScreen mainScreen] nativeBounds].size.width;
+        m_height = [[UIScreen mainScreen] nativeBounds].size.height;
+        // The rectangle returned by nativeBounds is always in a portrait orientation.
+        // If the app is landscape, width and height must be swapped.
+        if (IsLandscape())
+        {
+            AZStd::swap(m_width, m_height);
+        }
+        // Make sure that the window width does not exceed the one of the geometry.
+        if (geometry.m_width < m_width)
```

Comment:
This way we get a crop of full hd resolution that matches the device aspect. It resolves the issue with aspect and with LyShine, but it kills per-tier resolution tuning. We never get pixel perfect render for top devices with this code.